src/library/itunes/itunesxmlimporter.cpp

```diff
@@ -453,68 +482,27 @@ void ITunesXMLImporter::parsePlaylist(QSqlQuery& queryInsertToPlaylists,
                 if (key == "Playlist ID") {
                     readNextStartElement();
                     playlist_id = m_xml.readElementText().toInt();
-                    playlist_position = 1;
                     continue;
                 }
                 // Hide playlists that are system playlists
                 if (key == "Master" || key == "Movies" || key == "TV Shows" ||
                         key == "Music" || key == "Books" || key == "Purchased") {
-                    isSystemPlaylist = true;
+                    readNextStartElement();
+                    if (m_xml.name() == "true") {
+                        isSystemPlaylist = true;
+                    }
                     continue;
```

Comment:
Another minor fix:

This previously assumed that the presence of the `Master` key (and others as listed above) also implies the value `true`. While this works for iTunes/Apple Music, since their serializer seems to omit the key whenever it is `false`, this is another implementation detail that we shouldn't rely on. Not to mention that it breaks third-party tools that may generate `<key>Master</key><false/>`.